test/integration/smart_answer_flows/coronavirus_find_support_test.rb

```diff
@@ -0,0 +1,106 @@
+      add_response "no"
+      assert_current_node :afford_food?
+      add_response "no"
+
+      assert_current_node :get_food?
+      add_response "yes"
+      assert_current_node :self_employed?
+      add_response "yes"
+
+      assert_current_node :worried_about_work?
+      add_response "yes"
+      assert_current_node :have_somewhere_to_live?
+      add_response "no"
+      assert_current_node :have_you_been_evicted?
+      add_response "yes"
+      assert_current_node :mental_health_worries?
+      add_response "yes"
+      assert_current_node :nation?
+      add_response "england,scotland,wales,northern_ireland"
+      assert_current_node :results
+    end
+
+    should "show results for a user that cannot get food, is not self-employed, and has been made unemployed" do
+      assert_current_node :need_help_with?
+      add_response "feeling_unsafe"
+      assert_current_node :feel_safe?
+      add_response "no"
+      assert_current_node :afford_rent_mortgage_bills?
+      add_response "no"
+      assert_current_node :afford_food?
+      add_response "no"
+
+      assert_current_node :get_food?
+      add_response "no"
+      assert_current_node :able_to_go_out?
+      add_response "yes"
+      assert_current_node :self_employed?
+      add_response "no"
+      assert_current_node :have_you_been_made_unemployed?
+      add_response "yes_1"
+
+      assert_current_node :worried_about_work?
+      add_response "yes"
+      assert_current_node :have_somewhere_to_live?
+      add_response "no"
+      assert_current_node :have_you_been_evicted?
+      add_response "yes"
+      assert_current_node :mental_health_worries?
+      add_response "yes"
+      assert_current_node :nation?
+      add_response "england,scotland,wales,northern_ireland"
+      assert_current_node :results
+    end
+
+    should "show results for a user that can get food, is not self-employed, and has not been made unemployed" do
+      assert_current_node :need_help_with?
+      add_response "feeling_unsafe"
+      assert_current_node :feel_safe?
+      add_response "no"
+      assert_current_node :afford_rent_mortgage_bills?
+      add_response "no"
+      assert_current_node :afford_food?
+      add_response "no"
+
+      assert_current_node :get_food?
+      add_response "yes"
+      assert_current_node :self_employed?
+      add_response "no"
+      assert_current_node :have_you_been_made_unemployed?
+      add_response "no"
+      assert_current_node :are_you_off_work_ill?
+      add_response "no"
+
+      assert_current_node :worried_about_work?
+      add_response "yes"
+      assert_current_node :have_somewhere_to_live?
+      add_response "no"
+      assert_current_node :have_you_been_evicted?
+      add_response "yes"
+      assert_current_node :mental_health_worries?
+      add_response "yes"
+      assert_current_node :nation?
+      add_response "england,scotland,wales,northern_ireland"
+      assert_current_node :results
+    end
```

Comment:
I think there are three two-way branches in the logic, so I think that means six possible happy pathways through the flow. As there are so few specs, there should at least be one for each happy pathway.

Reply:
Thanks @reggieb 👍  - I was thinking that I'd missed a number of paths through the app.  I'll add more tests and appropriate logic.